### ISSUE TITLE
Add Helptext pattern

### ISSUE THREAD

**Issue by davidkudera** [adobe/spectrum-web-components#1704]
https://spectrum.adobe.com/page/text-field/#Help-text-(description-and-error-message)
https://github.com/adobe/spectrum-css/issues/1156

### Expected Behaviour

The new helptext component should be available in the library.

### Actual Behaviour

It's missing.

**Comment by Westbrook** [comment]
Great addition @davidkudera! This is blocked similar to #1643 by the work in #1666 and should be moving into focus for us in the near future.

**Comment by Westbrook** [comment]
Initial visuals only demo: https://help-text--spectrum-web-components.netlify.app/components/help-text/

Usage points:
- `sp-textfield`
- `sp-textfield[multiline]`
- `sp-picker`
- `sp-combobox` TK
- `sp-radio-group`
- `sp-field-group`
- others?

The two `*-group` elements work differently than the other as far as where the "helped" content lives and how to relate the two elements together, but I'd like to actual usage to be as close to the same as possible. 🤔 Will work on a proposal for that here soon.

**Comment by Westbrook** [comment]
This is either overly cheeky or really nice: https://webcomponents.dev/edit/CxGWNedMhM4g3yt442wW/src/index.ts

Need to think through the API here a little bit more, but the idea work well with Voice Over so far. Will review with NVDA (and hopefully JAWS) later today or tomorrow.

@jnurthen if we are to associate help text content via the `aria-describedby` attribute, should we also use `aria-hidden="true"` on the `<sp-help-text>` element so as to prevent the screen reader from voicing the content twice?

**Comment by Westbrook** [comment]
This covers usage in everything listed above, expect Picker, which requires being updated to a derivative of the Combobox pattern, IIUC, before it will accept described by content in this way, https://help-text--spectrum-web-components.netlify.app/components/help-text/#usage-1.

**Comment by jnurthen** [comment]
> @jnurthen if we are to associate help text content via the `aria-describedby` attribute, should we also use `aria-hidden="true"` on the `<sp-help-text>` element so as to prevent the screen reader from voicing the content twice?

No. You should not add aria-hidden on the `<sp-help-text>` element - the example looks good (from code inspection - haven't tried in screen readers but I would expect no issues)
